docs/integrations/ansible.md

```diff
@@ -1,19 +1,60 @@
 ---
-title: Conjur Ansible Role
+title: Ansible Integration
 layout: page
 section: integrations
 description: Conjur Integrations - Ansible Role
 ---
 
-The [Conjur Ansible Role](https://github.com/cyberark/ansible-role-conjur)
+## What is Ansible?
+Ansible is an automation language and automation engine that lets you describe end-to-end IT application environments w/ a “playbook.” Ansible’s simple, human-readable language allows orchestration of your application lifecycle no matter where it’s deployed.
```

Comment:
Do we want to spell out _with a “playbook.”_ here rather than shorthand it with the _w/_?  I don't think we use shorthand anywhere else and it looks out of place.